platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -651,6 +651,9 @@ private List<EventImpl> getStronglySeenInPreviousRound(final EventImpl event) {
         final List<EventImpl> judges = roundElections.findAllJudges();
         final long decidedRoundNumber = rounds.getElectionRoundNumber();
 
+        // Check for no judges or super majority conditions.
+        checkJudges(judges, decidedRoundNumber);
```

Comment:
looks good to me.
As an aside, i think this method can be better named - `writeMarkerFile` is probably more appropriate.

Reply:
I disagree, the writing of the marker files doesn't happen in all cases, only when the judges fail their checks.  I think check judges is the right name.